evalml/pipelines/pipeline_base.py

```diff
@@ -46,69 +40,63 @@
 class PipelineBase(ABC, metaclass=PipelineBaseMeta):
     """Base class for all pipelines."""
 
-    @property
-    @classmethod
-    @abstractmethod
-    def component_graph(cls):
-        """Returns list or dictionary of components representing pipeline graph structure
-
-        Returns:
-            list(str / ComponentBase subclass): List of ComponentBase subclasses or strings denotes graph structure of this pipeline
-        """
-
-    custom_hyperparameters = None
-    custom_name = None
     problem_type = None
 
-    def __init__(self, parameters, random_seed=0):
+    def __init__(self, component_graph,
+                 custom_name=None,
+                 parameters=None,
```

Comment:
Is there anything about the new design that made you set a default value for parameters? 

My intuition says "No" but just curious why the change was made.